frontend/app/src/components/home/MakeProposalModal.svelte

```diff
@@ -295,62 +331,106 @@
+                                id="chat_treasury"
+                                group="treasury"
+                                value={symbol}
+                                label={symbol}
+                                disabled={busy}
+                                checked={treasury === "SNS"}
+                                on:change={() => (treasury = "SNS")} />
+                            <Radio
+                                id="icp_treasury"
+                                group="treasury"
+                                value="ICP"
+                                label="ICP"
+                                disabled={busy}
+                                checked={treasury === "ICP"}
+                                on:change={() => (treasury = "ICP")} />
+                        </section>
+                        <section>
+                            <Legend label={$_("proposal.maker.recipientOwner")} required />
+                            <Input
+                                disabled={busy}
+                                invalid={recipientOwner.length > 0 && !recipientOwnerValid}
+                                maxlength={63}
+                                bind:value={recipientOwner}
+                                placeholder={$_("proposal.maker.enterRecipientOwner")} />
+                        </section>
+                        <section>
+                            <Legend
+                                label={$_("proposal.maker.recipientSubaccount")}
+                                rules={$_("proposal.maker.recipientSubaccountRules")} />
+                            <Input
+                                disabled={busy}
+                                invalid={!recipientSubaccountValid}
+                                maxlength={64}
+                                bind:value={recipientSubaccount}
+                                placeholder={$_("proposal.maker.enterRecipientSubaccount")} />
+                        </section>
+                        <section>
+                            <Legend
+                                label={$_("proposal.maker.amount")}
+                                rules={$_("proposal.maker.amountRules", { values: { token } })}
+                                required />
+                            <Input
+                                disabled={busy}
+                                invalid={amountText.length > 0 && !amountValid}
+                                minlength={1}
+                                maxlength={20}
+                                bind:value={amountText}
+                                placeholder={$_("proposal.maker.enterAmount", {
+                                    values: { token },
+                                })} />
+                        </section>
+                    </div>
+                {:else if selectedProposalType === "add_token"}
+                    <div>
+                        <section>
```

Comment:
What about the logo?

Reply:
We grab the logo from the metadata on the ledger. We could allow people to override it but that hasn't been needed on any ledger so far.